content/applications/services/helpdesk.rst

```diff
@@ -179,7 +178,8 @@ customer when a ticket reaches that specific stage in the pipeline. Likewise, ad
    <https://iap-services.odoo.com/iap/sms/pricing>`_ for additional information.
 
 To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
-the :guilabel:`→ (Internal Link)` icon to the right of the field to edit the chosen template.
+the :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon to the right of the field to edit the
```

Comment:
I know we have gone back and forth about this, but since the cog below is called "Actions", since that's what we see, you could make this "Internal Link" since that is what shows when you mouse over (just like the cog).  I think both would be fine.

Reply:
I talked to Sam about this and unless there is a specific aria label that gives the icon a different name (like the Actions icon) it is best to stick with the dev name, at least for now.